week2/index.html

```diff
@@ -0,0 +1,47 @@
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title >My First HTML</title>
+        <nav class="menu">
```

Comment:
Any content you want to display on your HTML page, should be part of &lt;body&gt; tag